dll/appcompat/shims/layer/versionlie.inl

```diff
@@ -31,14 +31,19 @@ BOOL WINAPI SHIM_OBJ_NAME(APIHook_GetVersionExW)(LPOSVERSIONINFOEXA lpOsVersionI
     return FALSE;
 }
 
+#ifndef STATUS_INVALID_PARAMETER
+#define STATUS_INVALID_PARAMETER 0xC000000DL
+#define STATUS_SUCCESS 0
+#endif
+
 /* We do not care about the actual type, FakeVersion will correctly handle it either way */
-BOOL WINAPI SHIM_OBJ_NAME(APIHook_RtlGetVersion)(LPOSVERSIONINFOEXA lpOsVersionInfo)
+DWORD WINAPI SHIM_OBJ_NAME(APIHook_RtlGetVersion)(LPOSVERSIONINFOEXA lpOsVersionInfo)
```

Comment:
Using NTSTATUS instead of DWORD wouldn't compile?

Reply:
indeed